package.json

```diff
@@ -3,6 +3,9 @@
   "version": "3.0.0",
   "description": "A generator for Yeoman",
   "keywords": [
+    "eslint",
+    "eslintplugin",
```

Comment:
We recommended for ESLint plugins to use these keywords. I don’t think it’s appropriate for this tool.

Reply:
Okay, we can add only `eslint` then.